### ISSUE TITLE
BART : How can I train and evaluate BART on CNN/DM dataset

### ISSUE THREAD

**Issue by Colanim** [huggingface/transformers#3004]
# ❓ Questions & Help

@sshleifer

I found examples for text summarization on CNN/DM dataset using BERT, but I couldn't find any example using BART.

**Are you going to add it later, or update the existing example to add BART, or it's not scheduled ?**

**Comment by BramVanroy** [comment]
BART has only just been implemented and is not part of release code yet. I'm sure that examples will be added later on, you just need a bit more patience.

**Comment by marmg** [comment]
How is this going? I would like to train BART for text summarization with my own dataset and I have no idea of what preprocessing is needed or what inputs the model needs. I would appreciate the help @sshleifer.

**Comment by sshleifer** [comment]
No training code yet, but we have an eval example. Its very new obviously, so make an issue and tag me if it breaks :)

**Comment by sshleifer** [comment]
https://github.com/huggingface/transformers/blob/b3e0a1cf0573a909f1769b5f1c2b7273faf36cf4/examples/summarization/bart/evaluate_cnn.py

**Comment by acarrera94** [comment]
The associated PR I opened has the training code. Just in case you want to test it out and run some experiments/give feedback. I based it on [this colab](https://colab.research.google.com/drive/1C4jEf0fnLiz6Xdx4TDz1OoO4BRCjCx1m) that I wrote.

**Comment by CarolLi** [comment]
I couldn't reach the accuracy in the paper with CNNDM dataset using the pre-trained model (facebook/bart-large-cnn).  Can anybody reproduce the accuracy properly?
